c++/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
IMO you might as well find a clang ppa and set 14 while you're at it. Shouldn't be much trouble.